src/components/EventSection/EventSection.jsx

```diff
@@ -0,0 +1,138 @@
+import PropTypes from "prop-types";
+
+const TicketCardContent = ({ onLinkClick }) => (
+    <>
+        <strong>Zarezerwuj swoje miejsce już teraz!</strong> Bilety na
+        wydarzenie kupisz{" "}
+        <a
+            href="https://bkb.pl/151648-edcca"
+            target="_blank"
+            rel="noopener noreferrer"
+            className={styles.eventLink}
+            onClick={() => onLinkClick("Kup bilety")}
+            aria-label="Kup bilety na Street Meeting Poland 2025"
+        >
+            tutaj
+        </a>
+        . <br />
+        <strong>Zyskaj 15% zniżki!</strong> Zapisz się do naszego newslettera i
+        odbierz kod rabatowy na bilet. Formularz zapisu znajdziesz na dole
+        strony!
+    </>
+);
+
+TicketCardContent.propTypes = {
+    onLinkClick: PropTypes.func.isRequired,
+};
+
+const SelectZoneCardContent = () => (
+    <>
+        <strong>Pokaż swój wyjątkowy pojazd!</strong> Masz unikalne auto? Nie
+        przegap szansy na jego prezentację <strong>na murawie stadionu</strong>{" "}
+        w strefie <strong>Select</strong>!
+    </>
+);
+
+const DriftTaxiCardContent = () => (
+    <>
+        Doświadcz prawdziwych motoryzacyjnych emocji! Widowiskowe pokazy driftu,
+        przejażdżki na fotelu pasażera i adrenalina na najwyższym poziomie! W
+        specjalnej strefie expo czekają na Ciebie najnowsze trendy motoryzacyjne
+        oraz renomowane marki z Polski i Europy.
+    </>
+);
+
+const EventSection = () => {
+    const handleLinkClick = (label) => {
+        if (window.gtag) {
+            window.gtag("event", "event_section_link_click", {
+                event_category: "EventSection",
+                event_label: label,
+            });
+        }
+    };
+
+    useEffect(() => {
+        const script = document.createElement("script");
+        script.type = "application/ld+json";
+        script.text = JSON.stringify({
+            "@context": "https://schema.org",
+            "@type": "Event",
+            name: "Street Meeting Poland 2025",
+            startDate: "2025-05-31T12:00:00+02:00",
+            endDate: "2025-05-31T23:00:00+02:00",
+            location: {
+                "@type": "Place",
+                name: "Polsat Plus Arena",
+                address: {
+                    "@type": "PostalAddress",
+                    streetAddress: "Pokoleń Lechii Gdańsk 1",
+                    addressLocality: "Gdańsk",
+                    postalCode: "80-560",
+                    addressCountry: "PL",
+                },
+            },
+            description:
+                "Przyjedź i odwiedź jedyne takie wydarzenie motoryzacyjne w Polsce.",
+            offers: {
+                "@type": "Offer",
+                url: "https://bkb.pl/151648-edcca",
+                priceCurrency: "PLN",
+                availability: "https://schema.org/InStock",
+            },
+        });
+        document.head.appendChild(script);
+
+        return () => {
```

Comment:
what this thing do?🤔 add script and immediately remove it? 

Can't it be script added in html header?